added for-loop exercise (#3)
wmvanvliet/gizmo: README.md

```

[Learn how to add a property](https://docs.python.org/3/library/functions.html#property)

## Exercise 6: Use a loop

> Add a `.spell()` method to your class that spells out its name, with dots between the 

Reviewer: ```suggestion
Add a `.spell()` method to your `Gizmo` class that uses the `print()` function to spell out its name, with dots between the 
```

> letters, using a loop.

Reviewer: ```suggestion
letters. Use a `for`-loop to implement this.
```

```

[Learn how to write a for loop in Python](https://docs.python.org/3.8/tutorial/controlflow.html#for-statements)

## Exercise 7 Using string formatting
Reading a bunch of files with only small differences in their filenames is a very common 
occurrence in any research code. Please extend the gizmo module with a method relative_path() 
that will return a list of files, including their relative path following the following pattern:
```text
" ./subjects/mock_recording_<subject_identifier>.rec"
``` 
where <subject_identifier> is any string. Subject identifiers will be passed to the method as a 
list of strings.

For example:
```python
>>> import gizmo

> >>> subject_identifiers = ["subject1","subject2"]

Reviewer: ```suggestion
>>> subject_identifiers = ["subject1", "subject2"]
```
